CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# Version 1.3.0-rc.1 Release Notes
```

Comment:
Why is this included? 🤔

Reply:
CI failed without it due to `get-vars.js`, I wanted to make sure it passed the checks.